### ISSUE TITLE
loss function matching between the paper and the project implementation

### ISSUE THREAD

**Issue by almogmor** [s3prl/s3prl#372]
I have a mismatch between the theoretical loss function to the way it's implemented in the project. 

my questions are: 
1) where is the loss function separation to layers?
2) where can I see the complete loss process (cos function, lambda parameter, the l_1 loss and the weighted sum. 
attaching screenshots. 

<img width="384" alt="image" src="https://user-images.githubusercontent.com/50709552/177283668-ff47ded2-69e0-4e33-86a1-ab51c2ba0f3e.png">

![image](https://user-images.githubusercontent.com/50709552/177283909-3a5ef4f2-532f-4747-bb8c-2db952c83d31.png)

Thanks!
Almog.

**Comment by leo19941227** [comment]
Hello!

I believe you might be tracing the irrelevant code, the loss function in the paper is about DistilHubert and you will need to go to the s3prl/pretrain/distiller folder. For more specific usage, @vectominist is the author of DistilHubert and he might give you more details.

**Comment by vectominist** [comment]
You can find them here:

https://github.com/s3prl/s3prl/blob/26113e9febbaaee6311f8c2cdd9475dc495acce9/s3prl/pretrain/distiller/pretrain_expert.py#L330-L337
